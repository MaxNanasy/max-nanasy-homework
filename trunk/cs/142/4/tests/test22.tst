main()
{
  var a : int;
  var b : int;
  const c = 11;
  get(a);
  get(a, b);
}
