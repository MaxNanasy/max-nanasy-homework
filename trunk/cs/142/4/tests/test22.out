***** Symbol Table Contents *****
N: int C: TYPE T: null
N: void C: TYPE T: null
N: short C: TYPE T: null
N: MAIN C: PROCEDURE T: void
